README.md

```diff
@@ -1,23 +1,67 @@
 
-Error wrapping behaves somewhat differently than existing packages. It relies on root errors that contain a full stack trace and wrap errors that contain a single stack frame. When errors from other packages are wrapped, a root error is automatically created before wrapping it with the new context. This allows `eris` to work with other error packages transparently and eliminates the need to manage stack traces manually. Unlike other packages, `eris` also works well with global error types by automatically updating stack traces during error wrapping.
+```golang
+// Create an error with eris.New() or eris.NewGlobal() if you want to maintain a set of global error values
+ErrBadRequest := eris.New("error bad request") // or eris.NewGlobal("error bad request")
 
-## Types of errors
+// Wrap the error with additional context
+err = eris.Wrap(ErrBadRequest, "received a request with no ID")
 
-`eris` is concerned with only three different types of errors: root errors, wrap errors, and external errors. Root and wrap errors are defined types in this package and all other error types are external or third-party errors.
+// Format the error using the default format or a custom format
+formattedErr := eris.ToJSON(err) // or eris.ToFormattedJSON(err, format)
 
-Root errors are created via [`eris.New`](https://godoc.org/github.com/rotisserie/eris#New) and [`eris.Errorf`](https://godoc.org/github.com/rotisserie/eris#Errorf). Generally, it's a good idea to maintain a set of root errors that are then wrapped with additional context whenever an error of that type occurs. Wrap errors represent a stack of errors that have been wrapped with additional context. Unwrapping these errors via [`eris.Unwrap`](https://godoc.org/github.com/rotisserie/eris#Unwrap) will return the next error in the stack until a root error is reached. [`eris.Cause`](https://godoc.org/github.com/rotisserie/eris#Cause) will also retrieve the root error.
+// Marshal it to JSON if necessary and print (your logger generally does this for you)
+fmt.Print(json.Marshal(formattedErr))
 
-When external error types are wrapped with additional context, a root error is first created from the original error. This creates a stack trace for the error and allows it to function with the rest of the `eris` package.
+// Or ideally, pass it directly to a logger
+logger.WithField("error", formattedErr).Error()
+```
 
 ## Wrapping errors with additional context
```

Comment:
Why is Unpack under wrapping? Also unpack summary needs to be changed. I think we need to put out how unpack can be used to get parts of error. I can do it in my PR if you like

Reply:
sure, i can leave that section for your PR